IOMC/RandomEngine/python/IOMC_cff.py

```diff
@@ -178,5 +178,14 @@
         initialSeed = cms.untracked.uint32(1234567),
         engineName = cms.untracked.string('HepJamesRandom')) )
 
-eras.phase2_timing.toModify(RandomNumberGeneratorService,
-                            trackTimeValueMapProducer = cms.PSet(initialSeed = cms.untracked.uint32(1234567), engineName = cms.untracked.string('HepJamesRandom') ) )
+eras.phase2_timing.toModify(
```

Comment:
@lgray - does this need to be in an era at all? (eg, is there harm in defining and not using?